### PR TITLE
Fixed bug when images was still showing placeholder

### PR DESCRIPTION
**Pull request by vladimirsm**

### CODE REVIEW

File: src/app/component/Image/Image.component.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ class Image extends Component {
     onImageLoad(img) {
         if (img.target.currentSrc.includes('.webp')
         || img.target.currentSrc.includes('.jpg')) this.setState({ isImageLoaded: true });
```
Comment (alfredsgenkins) [inline src/app/component/Image/Image.component.js]: Maybe nesting here? Maybe `{}` for better readability.

File: src/app/component/Image/Image.style.scss [hunk 2]
```diff
@@ -40,6 +40,10 @@
         mask: radial-gradient(ellipse, black 30%, transparent 61%, transparent 100%), radial-gradient(circle, black 24%, transparent 60%, transparent 100%);
     }
 
+    &_isLoaded {
+        background: transparent;
```
Comment (alfredsgenkins) [inline src/app/component/Image/Image.style.scss]: Not sure this is the best way. Can you please check, wether the homepage slider and cms blocks work and do not blink? There should be smooth transition from grey to image, without any blinks. Call me when you will be checking.

File: src/app/component/Image/Image.component.js [hunk 3]
```diff
@@ -53,6 +53,7 @@ class Image extends Component {
...
+        if (img.target.currentSrc.includes('.svg')) this.setState({ isPlacehodlerLoaded: true });
```
Comment (alfredsgenkins) [inline src/app/component/Image/Image.component.js]: Two setState in one function? Can you make it to one?

**Comment by vladimirsm** [comment]
This fixes #62

**Pull request merged**